server/config/database.js

```diff
@@ -27,7 +27,7 @@ export default () => ({
 
     review: {
         client: process.env.DATABASE_CLIENT,
-        connection: process.env.BACKUP_DATABASE,
+        connection: process.env.BACKUP_DATABASE_URL,
```

Comment:
I didn't know we had this. Also, why are you changing it? What's this for ?

Reply:
this was causing heroku review apps to not build

Reply:
left it